<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />

  <title>A flexible model for correlated count data, with application to multi-condition differential expression analyses of single-cell RNA sequencing data</title><!-- Begin Jekyll SEO tag v2.7.1 -->
<meta name="generator" content="Jekyll v3.9.5" />
<meta property="og:title" content="A flexible model for correlated count data, with application to multi-condition differential expression analyses of single-cell RNA sequencing data" />
<meta name="author" content="Yusha Liu, Peter Carbonetto, Michihiro Takahama, Adam Gruenbaum, Dongyue Xie, Nicolas Chevrier, Matthew Stephens" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Detecting differences in gene expression is an important part of single-cell RNA sequencing experiments, and many statistical methods have been developed for this aim. Most differential expression analyses focus on comparing expression between two groups (e.g., treatment vs. control). But there is increasing interest in multi-condition differential expression analyses in which expression is measured in many conditions, and the aim is to accurately detect and estimate expression differences in all conditions. We show that directly modeling single-cell RNA-seq counts in all conditions simultaneously, while also inferring how expression differences are shared across conditions, leads to greatly improved performance for detecting and estimating expression differences compared to existing methods. We illustrate the potential of this new approach by analyzing data from a single-cell experiment studying the effects of cytokine stimulation on gene expression. We call our new method “Poisson multivariate adaptive shrinkage”, and it is implemented in an R package available online at https://github.com/stephenslab/poisson.mash.alpha." />
<meta property="og:description" content="Detecting differences in gene expression is an important part of single-cell RNA sequencing experiments, and many statistical methods have been developed for this aim. Most differential expression analyses focus on comparing expression between two groups (e.g., treatment vs. control). But there is increasing interest in multi-condition differential expression analyses in which expression is measured in many conditions, and the aim is to accurately detect and estimate expression differences in all conditions. We show that directly modeling single-cell RNA-seq counts in all conditions simultaneously, while also inferring how expression differences are shared across conditions, leads to greatly improved performance for detecting and estimating expression differences compared to existing methods. We illustrate the potential of this new approach by analyzing data from a single-cell experiment studying the effects of cytokine stimulation on gene expression. We call our new method “Poisson multivariate adaptive shrinkage”, and it is implemented in an R package available online at https://github.com/stephenslab/poisson.mash.alpha." />
<link rel="canonical" href="https://emanuelealiverti.github.io/arxiv_rss/2024/05/28/AflexiblemodelforcorrelatedcountdatawithapplicationtomulticonditiondifferentialexpressionanalysesofsinglecellRNAsequencingdata.html" />
<meta property="og:url" content="https://emanuelealiverti.github.io/arxiv_rss/2024/05/28/AflexiblemodelforcorrelatedcountdatawithapplicationtomulticonditiondifferentialexpressionanalysesofsinglecellRNAsequencingdata.html" />
<meta property="og:site_name" content="Stat Arxiv of Today" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2024-05-28T00:00:00+00:00" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="A flexible model for correlated count data, with application to multi-condition differential expression analyses of single-cell RNA sequencing data" />
<script type="application/ld+json">
{"description":"Detecting differences in gene expression is an important part of single-cell RNA sequencing experiments, and many statistical methods have been developed for this aim. Most differential expression analyses focus on comparing expression between two groups (e.g., treatment vs. control). But there is increasing interest in multi-condition differential expression analyses in which expression is measured in many conditions, and the aim is to accurately detect and estimate expression differences in all conditions. We show that directly modeling single-cell RNA-seq counts in all conditions simultaneously, while also inferring how expression differences are shared across conditions, leads to greatly improved performance for detecting and estimating expression differences compared to existing methods. We illustrate the potential of this new approach by analyzing data from a single-cell experiment studying the effects of cytokine stimulation on gene expression. We call our new method “Poisson multivariate adaptive shrinkage”, and it is implemented in an R package available online at https://github.com/stephenslab/poisson.mash.alpha.","author":{"@type":"Person","name":"Yusha Liu, Peter Carbonetto, Michihiro Takahama, Adam Gruenbaum, Dongyue Xie, Nicolas Chevrier, Matthew Stephens"},"datePublished":"2024-05-28T00:00:00+00:00","mainEntityOfPage":{"@type":"WebPage","@id":"https://emanuelealiverti.github.io/arxiv_rss/2024/05/28/AflexiblemodelforcorrelatedcountdatawithapplicationtomulticonditiondifferentialexpressionanalysesofsinglecellRNAsequencingdata.html"},"url":"https://emanuelealiverti.github.io/arxiv_rss/2024/05/28/AflexiblemodelforcorrelatedcountdatawithapplicationtomulticonditiondifferentialexpressionanalysesofsinglecellRNAsequencingdata.html","headline":"A flexible model for correlated count data, with application to multi-condition differential expression analyses of single-cell RNA sequencing data","@type":"BlogPosting","dateModified":"2024-05-28T00:00:00+00:00","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link type="application/atom+xml" rel="alternate" href="https://emanuelealiverti.github.io/arxiv_rss/feed.xml" title="Stat Arxiv of Today" /><link rel="shortcut icon" type="image/x-icon" href="" />
  <link rel="stylesheet" href="/arxiv_rss/assets/css/main.css" />


<script>
MathJax = {
  tex: {
    inlineMath: [['$', '$'], ['\\(', '\\)']]
  },
  svg: {
    fontCache: 'global'
  }
};
</script>
<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>


</head>
<body a="light">
    <main class="page-content" aria-label="Content">
      <div class="w">
        <a href="/arxiv_rss/">..</a><article>
  <p class="post-meta">
    <time datetime="2024-05-28 00:00:00 +0000">05-28</time>
  </p>
  
  <h1>A flexible model for correlated count data, with application to multi-condition differential expression analyses of single-cell RNA sequencing data</h1>
  <br>Yusha Liu, Peter Carbonetto, Michihiro Takahama, Adam Gruenbaum, Dongyue Xie, Nicolas Chevrier, Matthew Stephens</h3>
  <br> [stat.ME,stat.AP]

  <p>Detecting differences in gene expression is an important part of single-cell RNA sequencing experiments, and many statistical methods have been developed for this aim. Most differential expression analyses focus on comparing expression between two groups (e.g., treatment vs. control). But there is increasing interest in multi-condition differential expression analyses in which expression is measured in many conditions, and the aim is to accurately detect and estimate expression differences in all conditions. We show that directly modeling single-cell RNA-seq counts in all conditions simultaneously, while also inferring how expression differences are shared across conditions, leads to greatly improved performance for detecting and estimating expression differences compared to existing methods. We illustrate the potential of this new approach by analyzing data from a single-cell experiment studying the effects of cytokine stimulation on gene expression. We call our new method “Poisson multivariate adaptive shrinkage”, and it is implemented in an R package available online at https://github.com/stephenslab/poisson.mash.alpha.</p>

<p><a href="https://arxiv.org/abs/2210.00697">Read more</a></p>

</article>

      </div>
    </main>
  </body>
</html>