<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />

  <title>Model-based Clustering of Zero-Inflated Single-Cell RNA Sequencing Data via the EM Algorithm</title><!-- Begin Jekyll SEO tag v2.7.1 -->
<meta name="generator" content="Jekyll v3.9.5" />
<meta property="og:title" content="Model-based Clustering of Zero-Inflated Single-Cell RNA Sequencing Data via the EM Algorithm" />
<meta name="author" content="Zahra AghahosseinaliShirazi, Pedro A. Rangel, Camila P. E. de Souza" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Biological cells can be distinguished by their phenotype or at the molecular level, based on their genome, epigenome, and transcriptome. This paper focuses on the transcriptome, which encompasses all the RNA transcripts in a given cell population, indicating the genes being expressed at a given time. We consider single-cell RNA sequencing data and develop a novel model-based clustering method to group cells based on their transcriptome profiles. Our clustering approach takes into account the presence of zero inflation in the data, which can occur due to genuine biological zeros or technological noise. The proposed model for clustering involves a mixture of zero-inflated Poisson or zero-inflated negative binomial distributions, and parameter estimation is carried out using the EM algorithm. We evaluate the performance of our proposed methodology through simulation studies and analyses of publicly available datasets." />
<meta property="og:description" content="Biological cells can be distinguished by their phenotype or at the molecular level, based on their genome, epigenome, and transcriptome. This paper focuses on the transcriptome, which encompasses all the RNA transcripts in a given cell population, indicating the genes being expressed at a given time. We consider single-cell RNA sequencing data and develop a novel model-based clustering method to group cells based on their transcriptome profiles. Our clustering approach takes into account the presence of zero inflation in the data, which can occur due to genuine biological zeros or technological noise. The proposed model for clustering involves a mixture of zero-inflated Poisson or zero-inflated negative binomial distributions, and parameter estimation is carried out using the EM algorithm. We evaluate the performance of our proposed methodology through simulation studies and analyses of publicly available datasets." />
<link rel="canonical" href="https://emanuelealiverti.github.io/arxiv_rss/2024/06/04/ModelbasedClusteringofZeroInflatedSingleCellRNASequencingDataviatheEMAlgorithm.html" />
<meta property="og:url" content="https://emanuelealiverti.github.io/arxiv_rss/2024/06/04/ModelbasedClusteringofZeroInflatedSingleCellRNASequencingDataviatheEMAlgorithm.html" />
<meta property="og:site_name" content="Stat Arxiv of Today" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2024-06-04T00:00:00+00:00" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Model-based Clustering of Zero-Inflated Single-Cell RNA Sequencing Data via the EM Algorithm" />
<script type="application/ld+json">
{"description":"Biological cells can be distinguished by their phenotype or at the molecular level, based on their genome, epigenome, and transcriptome. This paper focuses on the transcriptome, which encompasses all the RNA transcripts in a given cell population, indicating the genes being expressed at a given time. We consider single-cell RNA sequencing data and develop a novel model-based clustering method to group cells based on their transcriptome profiles. Our clustering approach takes into account the presence of zero inflation in the data, which can occur due to genuine biological zeros or technological noise. The proposed model for clustering involves a mixture of zero-inflated Poisson or zero-inflated negative binomial distributions, and parameter estimation is carried out using the EM algorithm. We evaluate the performance of our proposed methodology through simulation studies and analyses of publicly available datasets.","author":{"@type":"Person","name":"Zahra AghahosseinaliShirazi, Pedro A. Rangel, Camila P. E. de Souza"},"datePublished":"2024-06-04T00:00:00+00:00","mainEntityOfPage":{"@type":"WebPage","@id":"https://emanuelealiverti.github.io/arxiv_rss/2024/06/04/ModelbasedClusteringofZeroInflatedSingleCellRNASequencingDataviatheEMAlgorithm.html"},"url":"https://emanuelealiverti.github.io/arxiv_rss/2024/06/04/ModelbasedClusteringofZeroInflatedSingleCellRNASequencingDataviatheEMAlgorithm.html","headline":"Model-based Clustering of Zero-Inflated Single-Cell RNA Sequencing Data via the EM Algorithm","@type":"BlogPosting","dateModified":"2024-06-04T00:00:00+00:00","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link type="application/atom+xml" rel="alternate" href="https://emanuelealiverti.github.io/arxiv_rss/feed.xml" title="Stat Arxiv of Today" /><link rel="shortcut icon" type="image/x-icon" href="" />
  <link rel="stylesheet" href="/arxiv_rss/assets/css/main.css" />


<script>
MathJax = {
  tex: {
    inlineMath: [['$', '$'], ['\\(', '\\)']]
  },
  svg: {
    fontCache: 'global'
  }
};
</script>
<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>


</head>
<body a="light">
    <main class="page-content" aria-label="Content">
      <div class="w">
        <a href="/arxiv_rss/">..</a><article>
  <p class="post-meta">
    <time datetime="2024-06-04 00:00:00 +0000">06-04</time>
  </p>
  
  <h1>Model-based Clustering of Zero-Inflated Single-Cell RNA Sequencing Data via the EM Algorithm</h1>
  <br>Zahra AghahosseinaliShirazi, Pedro A. Rangel, Camila P. E. de Souza</h3>
  <br> [stat.ME,stat.AP]

  <p>Biological cells can be distinguished by their phenotype or at the molecular level, based on their genome, epigenome, and transcriptome. This paper focuses on the transcriptome, which encompasses all the RNA transcripts in a given cell population, indicating the genes being expressed at a given time. We consider single-cell RNA sequencing data and develop a novel model-based clustering method to group cells based on their transcriptome profiles. Our clustering approach takes into account the presence of zero inflation in the data, which can occur due to genuine biological zeros or technological noise. The proposed model for clustering involves a mixture of zero-inflated Poisson or zero-inflated negative binomial distributions, and parameter estimation is carried out using the EM algorithm. We evaluate the performance of our proposed methodology through simulation studies and analyses of publicly available datasets.</p>

<p><a href="https://arxiv.org/abs/2406.00245">Read more</a></p>

</article>

      </div>
    </main>
  </body>
</html>