<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />

  <title>Scalable Amortized GPLVMs for Single Cell Transcriptomics Data</title><!-- Begin Jekyll SEO tag v2.7.1 -->
<meta name="generator" content="Jekyll v3.9.5" />
<meta property="og:title" content="Scalable Amortized GPLVMs for Single Cell Transcriptomics Data" />
<meta name="author" content="Sarah Zhao, Aditya Ravuri, Vidhi Lalchand, Neil D. Lawrence" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Dimensionality reduction is crucial for analyzing large-scale single-cell RNA-seq data. Gaussian Process Latent Variable Models (GPLVMs) offer an interpretable dimensionality reduction method, but current scalable models lack effectiveness in clustering cell types. We introduce an improved model, the amortized stochastic variational Bayesian GPLVM (BGPLVM), tailored for single-cell RNA-seq with specialized encoder, kernel, and likelihood designs. This model matches the performance of the leading single-cell variational inference (scVI) approach on synthetic and real-world COVID datasets and effectively incorporates cell-cycle and batch information to reveal more interpretable latent structures as we demonstrate on an innate immunity dataset." />
<meta property="og:description" content="Dimensionality reduction is crucial for analyzing large-scale single-cell RNA-seq data. Gaussian Process Latent Variable Models (GPLVMs) offer an interpretable dimensionality reduction method, but current scalable models lack effectiveness in clustering cell types. We introduce an improved model, the amortized stochastic variational Bayesian GPLVM (BGPLVM), tailored for single-cell RNA-seq with specialized encoder, kernel, and likelihood designs. This model matches the performance of the leading single-cell variational inference (scVI) approach on synthetic and real-world COVID datasets and effectively incorporates cell-cycle and batch information to reveal more interpretable latent structures as we demonstrate on an innate immunity dataset." />
<link rel="canonical" href="https://emanuelealiverti.github.io/arxiv_rss/2024/05/08/ScalableAmortizedGPLVMsforSingleCellTranscriptomicsData.html" />
<meta property="og:url" content="https://emanuelealiverti.github.io/arxiv_rss/2024/05/08/ScalableAmortizedGPLVMsforSingleCellTranscriptomicsData.html" />
<meta property="og:site_name" content="Stat Arxiv of Today" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2024-05-08T00:00:00+00:00" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Scalable Amortized GPLVMs for Single Cell Transcriptomics Data" />
<script type="application/ld+json">
{"description":"Dimensionality reduction is crucial for analyzing large-scale single-cell RNA-seq data. Gaussian Process Latent Variable Models (GPLVMs) offer an interpretable dimensionality reduction method, but current scalable models lack effectiveness in clustering cell types. We introduce an improved model, the amortized stochastic variational Bayesian GPLVM (BGPLVM), tailored for single-cell RNA-seq with specialized encoder, kernel, and likelihood designs. This model matches the performance of the leading single-cell variational inference (scVI) approach on synthetic and real-world COVID datasets and effectively incorporates cell-cycle and batch information to reveal more interpretable latent structures as we demonstrate on an innate immunity dataset.","mainEntityOfPage":{"@type":"WebPage","@id":"https://emanuelealiverti.github.io/arxiv_rss/2024/05/08/ScalableAmortizedGPLVMsforSingleCellTranscriptomicsData.html"},"headline":"Scalable Amortized GPLVMs for Single Cell Transcriptomics Data","dateModified":"2024-05-08T00:00:00+00:00","datePublished":"2024-05-08T00:00:00+00:00","url":"https://emanuelealiverti.github.io/arxiv_rss/2024/05/08/ScalableAmortizedGPLVMsforSingleCellTranscriptomicsData.html","author":{"@type":"Person","name":"Sarah Zhao, Aditya Ravuri, Vidhi Lalchand, Neil D. Lawrence"},"@type":"BlogPosting","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link type="application/atom+xml" rel="alternate" href="https://emanuelealiverti.github.io/arxiv_rss/feed.xml" title="Stat Arxiv of Today" /><link rel="shortcut icon" type="image/x-icon" href="" />
  <link rel="stylesheet" href="/arxiv_rss/assets/css/main.css" />


<script>
MathJax = {
  tex: {
    inlineMath: [['$', '$'], ['\\(', '\\)']]
  },
  svg: {
    fontCache: 'global'
  }
};
</script>
<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>


</head>
<body a="light">
    <main class="page-content" aria-label="Content">
      <div class="w">
        <a href="/arxiv_rss/">..</a><article>
  <p class="post-meta">
    <time datetime="2024-05-08 00:00:00 +0000">05-08</time>
  </p>
  
  <h1>Scalable Amortized GPLVMs for Single Cell Transcriptomics Data</h1>
  <br>Sarah Zhao, Aditya Ravuri, Vidhi Lalchand, Neil D. Lawrence</h3>
  <br> [stat.ML,stat.AP]

  <p>Dimensionality reduction is crucial for analyzing large-scale single-cell RNA-seq data. Gaussian Process Latent Variable Models (GPLVMs) offer an interpretable dimensionality reduction method, but current scalable models lack effectiveness in clustering cell types. We introduce an improved model, the amortized stochastic variational Bayesian GPLVM (BGPLVM), tailored for single-cell RNA-seq with specialized encoder, kernel, and likelihood designs. This model matches the performance of the leading single-cell variational inference (scVI) approach on synthetic and real-world COVID datasets and effectively incorporates cell-cycle and batch information to reveal more interpretable latent structures as we demonstrate on an innate immunity dataset.</p>

<p><a href="https://arxiv.org/abs/2405.03879">Read more</a></p>

</article>

      </div>
    </main>
  </body>
</html>